docs/proposals/permission-level-degradation.md

```diff
@@ -0,0 +1,188 @@
+# Bulward Permission Level Degradation
+
+This document extends on the initial Bulward concept to enable degradation of user defined roles.
+
+## Background
+
+After installing bulward organization owners will have very limited permissions. Essentially they will only be allowed to interact with the Bulward API -> e.g. creating new Projects, organize users and create sub-roles.
+
+In order to actually grant an organization rights to do something, these rights need to be explicitly assigned to an Organization.
+New Operators may ship `OrganizationRoleTemplates` to setup default roles to interact with it.
+
+This is all fine, but it gets complicated when Organization Owners define their own sub-roles.
+
+While granting new permissions is uncomplicated, removing permissions is less easy, because new roles utilizing these permissions might have been created.
+This concept deals with the necessary permission/role cleanup, by introducing new theoretical concepts and extending the KubeCarrier API.
+
+## Concepts
+
+### Max Permission Level (Organization/Project)
+
+Organizations and Projects have both a maximum permission level that cannot be exceeded within.
```

Comment:
How is this maximum permission configured? In the `Organization` / `Project` resource or?